Makefile

```diff
@@ -178,17 +197,17 @@ check-dirty: ## Verifies that source tree is not dirty
 
 .PHONY: unit-tests
 unit-tests: ## Performs unit tests.
-	@$(MAKE) local-$@ DEST=$(ARTIFACTS) TARGET_ARGS="--allow security.insecure"
+	@$(MAKE) local-$@ DEST=$(ARTIFACTS) TARGET_ARGS="--allow security.insecure" PLATFORM=linux/amd64
```

Comment:
this (and similar changes): force single-platform to avoid running it for ARM for now (we can revisit unit-tests later, but most of others are single-platform sufficient)